tests/data/seasonet/data.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import hashlib
+import os
+import shutil
+
+import numpy as np
+import rasterio
+from rasterio.crs import CRS
+from rasterio.transform import Affine
+
+np.random.seed(0)
+
+meta = {
+    "driver": "GTiff",
+    "nodata": None,
+    "crs": CRS.from_epsg(32632),
+    "transform": Affine(10.0, 0.0, 664800.0, 0.0, -10.0, 5342400.0),
+    "compress": "zstd",
```

Comment:
Just want to confirm that the original SeasoNet files are also compressed with ZSTD

Reply:
Yes, they are